src/korean.js

```diff
@@ -93,6 +96,12 @@ class Korean {
     } // end of first else
   } // end of presentWord function
 
+  doPresentContinuous(word) {
+    let wordLength = word.length;
+    let conjugate = word.substring(0, wordLength-1);
+    return `${conjugate}고있어`;
```

Comment:
Also, since this function is so simple, it might not warrant a `doPresentContinuous` function. You could directly return in the PresentContinuous case. The reason why doPresent exist is because that function was reused in the past tense method.